components/brave_rewards/resources/page/storage.ts

```diff
@@ -46,6 +46,7 @@ export const defaultState: Rewards.State = {
     shouldAllowAdsSubdivisionTargeting: true,
     adsUIEnabled: false,
     adsIsSupported: false,
+    needUpdateBrowserToSeeAds: false,
```

Comment:
This should work in the upgrade case, since a value of `undefined` will be treated just the same as `false`, but I would suggest testing that before merging.

Reply:
tested, upgrade works fine